src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,193 @@
+        LayerTestsUtils::LayerTestsCommon::Run();
+    }
+
+    void RunBenchmark(const std::string& name,
+                      const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+                      const int numAttempts = kDefaultNumberOfAttempts) {
+        if (!name.empty()) {
+            RunBenchmark({name}, warmupTime, numAttempts);
+        } else {
+            RunBenchmark({}, warmupTime, numAttempts);
+        }
+    }
+
+    // NOTE: Validation is ignored because we are interested in benchmarks results.
+    //       In future the validate method could check if new benchmark results are not worse than previous one
+    //       (regression test), and in case of any performance issue report it in PR.
+    void Validate() override {
+    }
+
+protected:
+    void Infer() override {
+        this->inferRequest = this->executableNetwork.CreateInferRequest();
+        this->ConfigureInferRequest();
+
+        std::map<std::string, long long> results_us{};
+        for (const auto& node_type_name : bench_node_type_names_) {
+            results_us[node_type_name] = {};
+        }
+
+        // Warmup
+        auto warm_current = std::chrono::steady_clock::now();
+        const auto warm_end = warm_current + warmup_time_;
+        while (warm_current < warm_end) {
+            this->inferRequest.Infer();
+            warm_current = std::chrono::steady_clock::now();
+        }
+
+        // Benchmark
+        for (int i = 0; i < num_attempts_; ++i) {
+            this->inferRequest.Infer();
+            const auto& perf_results = LayerTestsUtils::LayerTestsCommon::inferRequest.GetPerformanceCounts();
+            for (auto& res : results_us) {
+                const std::string node_type_name = res.first;
+                long long& time = res.second;
+                auto found_profile = std::find_if(perf_results.begin(), perf_results.end(),
+                    [&node_type_name](const InferenceEngine::InferenceEngineProfileInfo& profile) {
+                        return profile.layer_type == node_type_name;
+                    });
+                IE_THROW() << "Cannot find operator by node type: " << node_type_name;
+                time += found_profile->second.realTime_uSec;
+            }
+        }
+
+        long long total_us = 0;
+        for (auto& res : results_us) {
+            const std::string node_type_name = res.first;
+            long long& time = res.second;
+            time /= num_attempts_;
+            total_us += time;
+            std::cout << std::fixed << std::setfill('0') << node_type_name << ": " << time << " us\n";
+        }
+        std::cout << std::fixed << std::setfill('0') << "Total time: " << total_us << " us\n";
+    }
+
+private:
+    std::vector<std::string> bench_node_type_names_;
+    std::chrono::milliseconds warmup_time_;
+    int num_attempts_;
+};
+
+}  // namespace LayerTestsDefinitions
+
+namespace ov {
```

Comment:
the same comments as for the `LayerTestCommon`

Reply:
Fixed